src/main/java/org/molgenis/vcf/report/mapper/info/VepInfoMetadataMapper.java

```diff
@@ -56,38 +59,32 @@ protected InfoMetadata mapNestedId(String id) {
       case "CLIN_SIG":
       case "FLAGS":
         infoMetadata =
-            new InfoMetadata(
-                id,
-                new Number(Type.OTHER, null, '&'),
-                InfoMetadata.Type.STRING,
-                id,
-                null,
-                null,
-                Collections.emptyList());
+            InfoMetadata.builder()
+                .id(id)
+                .number(new Number(Type.OTHER, null, '&'))
+                .type(CompoundMetadata.Type.STRING)
+                .description(id)
+                .build();
         break;
       case "PHENO":
       case "SOMATIC":
         infoMetadata =
-            new InfoMetadata(
-                id,
-                new Number(Type.OTHER, null, '&'),
-                InfoMetadata.Type.INTEGER,
-                id,
-                null,
-                null,
-                Collections.emptyList());
+            InfoMetadata.builder()
+                .id(id)
+                .number(new Number(Type.OTHER, null, '&'))
+                .type(CompoundMetadata.Type.INTEGER)
+                .description(id)
+                .build();
         break;
       case "STRAND":
       case "HGNC_ID":
         infoMetadata =
-            new InfoMetadata(
-                id,
-                new Number(Type.NUMBER, 1, ','),
-                InfoMetadata.Type.INTEGER,
-                id,
-                null,
-                null,
-                Collections.emptyList());
+            InfoMetadata.builder()
+                .id(id)
+                .number(new Number(Type.NUMBER, 1, ','))
```

Comment:
Why the separator if number == 1?
And the comma cannot be the separator for a nested metadata item right, since it the separator for the 'main' VEP values?

Reply:
Technical reason: Lombok puts all arguments in the constructor by default. I can add a builder and use it everywhere to make it clear that no separator is required in case of number=1.